### PR TITLE
fix(nudge): update to use border radius style variable

### PR DESCRIPTION
**Pull request by JohnnyB83**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @JohnnyB83, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

File: src/components/nudge/Nudge.scss [hunk 1]
```diff
@@ -9,7 +9,7 @@
     padding: 25px;
     background-color: $white;
     border: solid 1px $bdl-gray-02;
-    border-radius: 30px;
+    border-radius: 4px;
```
Comment (alexkrolick) [inline src/components/nudge/Nudge.scss]: https://github.com/box/box-ui-elements/blob/master/src/styles/constants/_layout.scss#L9

Comment (JohnnyB83) [reply src/components/nudge/Nudge.scss]: thanks, resolved in latest commit

**Comment by laughnan** [comment]
this probably has snapshot implications for storybook that we should update.

**Pull request merged**